wp-parsely.php

```diff
@@ -139,6 +139,18 @@ function parsely_rest_api_init(): void {
 	$endpoint->run();
 }
 
+require __DIR__ . '/src/blocks/recommendations/class-recommendations-block.php';
+/**
+ * Register the Recommendations Block.
+ *
+ * @return void
+ */
+function init_recommendations_block() {
+	$recommendations_block = new Recommendations_Block();
+	$recommendations_block->run();
+}
+add_action( 'init', __NAMESPACE__ . '\\init_recommendations_block' );
```

Comment:
Just shuffles the order around so the add_action() call comes right before the DocBlock of the callback. This order makes it easier to understand what parameters might be going into the callback.

```suggestion
require __DIR__ . '/src/blocks/recommendations/class-recommendations-block.php';

add_action( 'init', __NAMESPACE__ . '\\init_recommendations_block' );
/**
 * Register the Recommendations Block.
 *
 * @return void
 */
function init_recommendations_block(): void {
	$recommendations_block = new Recommendations_Block();
	$recommendations_block->run();
}
```

Adds `: void` as well.